src/components/ChartFunctional.js

```diff
@@ -1,15 +1,13 @@
-import React,  {useState, useEffect } from 'react';
+import React from 'react';
 import { Line } from "react-chartjs-2";
 import moment from "moment";
 
-export default ({values: { balances }}) => {
```

Comment:
please name it as Chart:
export const Chart = (...

let's not use export defaults, things should have names

Reply:
components always should have names? 
is it better practice than my no-name?

Reply:
Using names allows you to disconnect class name from file name.
Also it reduces number of mind jumps, as you don't need to think about filename when u see default import, u just have a name before you.
Plus you don't need to think if thing you import have default import or named one if all are done in same manner.
However, you may have other arguments for using default imports.